### DISCUSSION TITLE
Multiple Form Blocks on single screen

### DISCUSSION THREAD

**Discussion by LifeofGlen** [Budibase/budibase#14326] [Help] [answered]
It's possible I am missing something obvious, but is there an easy way to have different form blocks display adjacent to each other on a single screen instead of always underneath each other? actually just want to have any components be adjacent to a form block instead of beneath it.

Somehow I'm unable to get them to visually be adjacent to each other.

any pointers/help appreciated.

**Comment by melohagan** [comment, marked as answer]
Hey @LifeofGlen 

This is easy to do with [Sections](https://docs.budibase.com/docs/using-sections)

![Screenshot 2024-08-09 at 13 35 46](https://github.com/user-attachments/assets/07a99a7e-b662-480d-aa4c-913ae775143c)

**Reply by LifeofGlen** [reply]
Thanks, knew i was missing something obvious :)
